app/controllers/todos_controller.rb

```diff
@@ -20,7 +21,7 @@ def show
 
   # POST /todos
   def create
-    @todo = Todo.new(todo_params)
+    @todo = Todo.new(todo_params.merge!(owner_id: current_owner))
```

Comment:
The IDs of newly created todos are now issued by the service. The FE doesn't include the `id` or `owner_id` fields anymore.